scripts/README.md

```diff
@@ -0,0 +1,36 @@
+## Syncing downstream OLM
```

Comment:
Super minor nit just for markdown linting purposes. Thanks for getting the ball rolling on this though - introducing directory-level READMEs is always super helpful.

```suggestion
# Syncing downstream OLM
```